lib/csv_import.rb

```diff
@@ -7,6 +7,10 @@ def csv_import(filepath)
     string = File.read(filepath)
     require 'csv'
 
+    # sanitize input
+    string = string.encode('UTF-8', 'binary',
+                           invalid: :replace, undef: :replace, replace: '')
```

Comment:
Did you check to make sure [this](https://stackoverflow.com/questions/11375342/stringencode-not-fixing-invalid-byte-sequence-in-utf-8-error) wasn't an issue? I'm guessing you created dummy CSV files for manual testing - if this didn't show up then that's fine but it came up while I was searching for stuff related to this issue.